components/molecule/buttonGroup/src/index.js

```diff
@@ -1,54 +1,100 @@
+import {DESIGNS, SIZES} from '@s-ui/react-atom-button/src/config.js'
```

Comment:
😭 We're having problems with this, because it's directly requiring the config file from the `src` folder. (Should be the lib folder?)

Reply:
😇